### PR TITLE
[Method References 2] LPS-171023 Convert method reference to ServiceTrackerMap in class IndexResourceImpl(module portal-workflow-metrics-rest-impl) 

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:workflow-metrics

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts(12/26 10:37)

**Comment by W2jx** [comment]
no conflicts(12/27 9:21)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-api/src/main/java/com/liferay/portal/workflow/metrics/search/index/name/WorkflowMetricsIndexNameBuilder.java [hunk 1]
```diff
@@ -14,10 +14,12 @@
...
-import com.liferay.portal.search.index.IndexNameBuilder;
-
 /**
  * @author Rafael Praxedes
  */
-public interface WorkflowMetricsIndexNameBuilder extends IndexNameBuilder {
+public interface WorkflowMetricsIndexNameBuilder {
+
+	public String getIndexName(
+		long companyId, WorkflowMetricsIndexEntityName indexEntityName);
```
Comment (tinatian) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-api/src/main/java/com/liferay/portal/workflow/metrics/search/index/name/WorkflowMetricsIndexNameBuilder.java:23]: @W2jx `WorkflowMetricsIndexEntityName indexEntityName`, please rename the parameter to "workflowMetricsIndexEntityName"

**Comment by tinatian** [comment]
@W2jx Please see my comment and resend.

Thanks.
Tina

**Pull request closed without merging**